app/controllers/payments_controller.rb

```diff
@@ -62,12 +75,15 @@ def webhook
     hips_id    = resource['id']
 
     payment = Payment.find(payment_id)
-    payment.status = Payment.order_to_payment_status(resource['status'])
-    payment.save
+    payment.update(status: Payment.order_to_payment_status(resource['status']))
+
+    # If user can pay member fee they are 1) already a member, or,
+    # have an accepted application
+    payment.user.update(member: true)
 
```

Comment:
This should also be the moment when the membership_number is issued. At the moment, that still happens when the application is accepted.